scalafmt-tests/src/test/resources/test/JavaDoc.stat

```diff
@@ -3218,3 +3218,33 @@ object a {
    */
   val a = 1
 }
+<<< preserve indentation in code blocks
+docstrings.style = Asterisk
```

Comment:
you need to do this for all styles, otherwise it's not fully tested. AsteriskSpace is the tricky one. i would also recommend switching styles; that is, write the original in, say, SpaceAsterisk but ask to reformat as AsteriskSpace. and make sure it still works and doesn't lead to non-idempotence.

Reply:
good point.
added